app/util/mqttClient.js

```diff
@@ -134,7 +135,14 @@ export function changeTopics(settings, actionContext) {
 
 export function startMqttClient(settings, actionContext) {
   const options = settings.options || [{}];
-  const topics = options.map(option => getTopic(option, settings));
+  let topics = options.map(option => getTopic(option, settings));
+  // If a general topic is returned, at least limit it to instances feed ids
+  if (topics[0] === '/gtfsrt/vp/+/+/+/+/+/+/+/+/+/+/+/+/+/+/+/#') {
+    const { feedIds } = actionContext.config;
+    topics = feedIds.map(
+      feedId => `/gtfsrt/vp/${feedId}/+/+/+/+/+/+/+/+/+/+/+/+/+/+/#`,
+    );
+  }
```

Comment:
This topic code is a bit too specific to this one MQTT broker. Couldn't it be possible to somehow define in the Map.js startClient that we should subscribe to these topics?